package.json

```diff
@@ -1,47 +1,49 @@
 {
   "name": "nextjs-project",
   "version": "1.0.0",
+  "private": "true",
   "description": "Next.js starter kit for Wunder projects.",
+  "keywords": [],
+  "homepage": "https://github.com/wunderio/nextjs-project#readme",
+  "bugs": {
+    "url": "https://github.com/wunderio/nextjs-project/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/wunderio/nextjs-project.git"
+  },
+  "author": "",
   "main": "index.js",
   "directories": {
     "doc": "docs"
   },
-  "dependencies": {
-    "next": "^12.1.6",
-    "react": "^18.0.0",
-    "react-dom": "^18.0.0"
-  },
-  "devDependencies": {
-    "@types/node": "^16.11.7",
-    "@types/react": "^17.0.43",
-    "@typescript-eslint/eslint-plugin": "^5.18.0",
-    "@typescript-eslint/parser": "^5.18.0",
-    "eslint": "^8.16.0",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-jsx-a11y": "^6.5.1",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-react": "^7.29.4",
-    "eslint-plugin-react-hooks": "^4.4.0",
-    "prettier": "^2.6.2",
-    "typescript": "^4.6.3"
-  },
   "scripts": {
-    "dev": "next dev",
     "build": "next build",
-    "start": "next start",
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "next dev",
     "lint": "eslint .",
-    "lint:fix": "eslint . --fix"
+    "lint:fix": "eslint . --fix",
+    "start": "next start",
+    "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/wunderio/nextjs-project.git"
+  "dependencies": {
+    "next": "^13.0.0",
```

Comment:
13.0.2 is out already :-)

Reply:
It's neverending 😭  Heh, it's ok though - developers should update anyway when this template is used for a new project.

Reply:
It is neverending indeed! :smile:

Reply:
You might want to add a comment about this in the README though, my experience is that many developers are afraid to do those kinds of updates (and often too busy)

Reply:
Developers are in charge of all the package versions anyway, starting from the day they click that "Use this template" button, right? Or is there an assumption that new versions could be pulled down from a starter like this?

Reply:
As an outside possible user of this repo I think it could be helpful if the README states that the first step after clicking "Use this template" is to update all the modules.

Reply:
Yeah, that would be good. As we don't have resources to "chase head" with the packages. PR's from outside are also appreciate 😊

Reply:
I just made https://github.com/wunderio/nextjs-project/pull/224 if either of you fancy taking a look :)